### DISCUSSION TITLE
ae图层渲染失败

### DISCUSSION THREAD

**Discussion by Xiaolianmao** [Tencent/libpag#1923] [Q&A] [answered]
环境:
macOS Ventura13.4 | AE  23.6.0(Build 53) | PAGViewer 4.3.8.105(beta)

问题:
矩形工具渐变色图层,PAG 导出失败
[bug.zip](https://github.com/Tencent/libpag/files/13484838/bug.zip)

**Comment by kevingpqi123** [comment]
文件夹下面不是有 pag 文件吗，导出失败指的是啥

**Reply by Xiaolianmao** [reply]
正确的一共为两个图层,底涂层没有渲染出来

**Reply by Xiaolianmao** [reply]
有提供ae的工程文件 你可以下载来看看

**Comment by domchen** [comment]
两个图层其实都导出了，出问题的地方是导出的文件总是下面那个图层会引用上面的那个图层作为tackMatte遮罩，但是看源文件里并没有设置这个遮罩。我们排查看看。

**Reply by Xiaolianmao** [reply]
感谢回复. 其实pag在遮罩上发生的问题还是比较频繁的了...时不时会出现图层漏渲染的是事情, 把工程重新做一遍又正常了.很奇怪.

**Comment by jinwuwu001** [comment, marked as answer]
该问题已修复，可从pag官网下载最新版PAGViewer，然后安装最新版AE导出插件
https://pag.io/
